cmd/destroy/destroy_test.go

```diff
@@ -905,19 +955,22 @@ func TestDestroyWithForceOptionAndFailedCommands(t *testing.T) {
 	if err != nil {
 		t.Fatal("could not create fake k8s client")
 	}
-	cmd := &destroyCommand{
-		getManifest:       getFakeManifest,
-		secrets:           &secretHandler,
-		executor:          executor,
-		nsDestroyer:       destroyer,
-		k8sClientProvider: k8sClientProvider,
-		configMapHandler:  newConfigmapHandler(fakeClient),
+
+	ld := localDestroyCommand{
+		&localDestroyAllCommand{
+			configMapHandler:  newConfigmapHandler(fakeClient),
+			nsDestroyer:       destroyer,
+			executor:          executor,
+			k8sClientProvider: k8sClientProvider,
+			secrets:           &secretHandler,
+		},
+		fakeManifest,
 	}
 
-	err = cmd.runDestroy(ctx, opts)
+	err = ld.runDestroy(ctx, opts)
 
 	assert.Error(t, err)
-	assert.ElementsMatch(t, fakeManifest.Destroy, executor.executed)
+	assert.ElementsMatch(t, fakeManifest.Destroy.Commands, executor.executed)
 	assert.True(t, destroyer.destroyed)
 	assert.True(t, destroyer.destroyedVolumes)
 
```

Comment:
Can we add a test for `destroy all` and another one for `destroy remote`?

Reply:
We need to fix `destroy all` with the fix in master so I will work on test `destroy remote`